CHANGELOG.md

```diff
@@ -7406,6 +7425,7 @@ Swift 1.0
 [SE-0227]: <https://github.com/apple/swift-evolution/blob/master/proposals/0227-identity-keypath.md>
 [SE-0228]: <https://github.com/apple/swift-evolution/blob/master/proposals/0228-fix-expressiblebystringinterpolation.md>
 [SE-0230]: <https://github.com/apple/swift-evolution/blob/master/proposals/0230-flatten-optional-try.md>
+[SE-O235]: <https://github.com/apple/swift-evolution/blob/master/proposals/0235-add-result.md>
```

Comment:
Typo (LATIN CAPITAL LETTER O should be DIGIT ZERO):

```diff
-[SE-O235]: <https://github.com/apple/swift-evolution/blob/master/proposals/0235-add-result.md>
+[SE-0235]: <https://github.com/apple/swift-evolution/blob/master/proposals/0235-add-result.md>
```

Reply:
Oy! Fixed in #22879.